slackin/README.md

```diff
@@ -0,0 +1,13 @@
+# SlackIn
+
+Purpose: Automated (open) enrollment for the SPIFFE Slack organization
+
+Primary Owner: @drrt
+
+Users/Groups with Access: Unknown
+
+Billing: Paid, Unknown
```

Comment:
@drrt is this paid in perpetuity or is it an ongoing cost? Do you know what card is attached to it?

Reply:
This is hosted in Heroku, i added @anvega as a co-admin a while back, and i'm personally paying $4/mo for it to enable their TLS features :D

Reply:
Ok awesome. I'll get this updated... and part of this work will also be alleviating you of the burden.

Sounds like I owe you a lunch or two :)